Code/GraphMol/FileParsers/testPropertyLists.cpp

```diff
@@ -170,16 +170,16 @@ one n/a three
     suppl.setProcessPropertyLists(false);
     std::unique_ptr<RDKit::ROMol> m(suppl[0]);
     REQUIRE(m);
-    CHECK(m->hasProp("atom.prop.AtomSymbol"));
-    CHECK(!m->getAtomWithIdx(0)->hasProp("AtomSymbol"));
+    CHECK(m->hasProp("atom.prop.AtomLabel"));
```

Comment:
Sorry, I did make this change as you suggested.  It seems to have been lost somewhere.